### DISCUSSION TITLE
Getting Stuck at adb remount

### DISCUSSION THREAD

**Discussion by mllockwood** [pgale/lineage_15.1_Installation_TSV#3] [Q&A] [answered]
I am trying to modify the MAC address of the ThinkSmart device since I have several of these on my network.  I have successfully performed this same update numerous times on the regular Android 8 and 11 versions.  But, when I follow the same steps to perform the file modification, the ADB  Fastboot is getting stuck at "adb remount".   I issue that command and nothing happens.  The ThinkSmart device begins to get noticeably sluggish and the touchscreen starts to not respond.  Eventually, I have to power cycle the device.  Has anyone else run in to this problem during this part of the process?

**Comment by mllockwood** [comment]
Additionally, I am getting the same results on 2 additional ThinkSmart devices.  So, I do not believe it's device specific.

**Reply by pgale** [reply]
thanks for the report - I've not tried changing the mac address myself and those instructions were from the devs on Discord. I'll add it to to the list to ask.

**Comment by mllockwood** [comment, marked as answer]
This may be a non-issue actually.  I have now installed LineageOS on my 3rd ThinkSmart device.  The devices each have had a unique MAC address.  Based on this, I believe that the instructions regarding changing MAC addresses for this installation can be removed.  Someone else may want to confirm my findings though.  I am about to load LineageOS on 4 more of these devices and will report my findings.

**Reply by pgale** [reply]
Thanks. All five of my devices now reinstalled with Lineage and have unique mac addresses but they did all have other versions on first which someone mentioned helps. Not sure how though. Seems those instructions don't work anyway.

**Reply by Elder-HVAC-Man** [reply]
I am in the process of installing on my 3rd device and all is going great.  Appreciate all of the work from everyone for this.

**Comment by pgale** [comment]
@mllockwood have you successfully flashed your other devices now?

**Reply by mllockwood** [reply]
I did.  No issues with DHCP and I did not need to do the modifications.  I believe those set of instructions are only applicable to Android not LineageOS.

**Comment by PiotrMachowski** [comment]
I have the same problem with `adb remount` - I would like to automatically enable adb over the network, is there a different way to achieve it? (other than executing a script on device boot which requires remount)
